src/Symfony/Component/Console/Logger/ConsoleLogger.php

```diff
@@ -106,7 +106,7 @@ private function interpolate(string $message, array $context): string
             if (null === $val || \is_scalar($val) || $val instanceof \Stringable) {
                 $replacements["{{$key}}"] = $val;
             } elseif ($val instanceof \DateTimeInterface) {
-                $replacements["{{$key}}"] = $val->format(\DateTime::RFC3339);
+                $replacements["{{$key}}"] = $val->format(\DateTimeImmutable::RFC3339);
```

Comment:
for all consts, we'd better reference the interface all the time

Reply:
Ok should be fixed with my last commit